### DISCUSSION TITLE
Disk plot Testing Questions

### DISCUSSION THREAD

**Discussion by WillBVT** [Chia-Network/bladebit#208] [Q&A] [answered]
Hi Guys,

So I'm trying to test out the new diskplotter and use Bladebit for the first time as I only have 32GB of RAM. I'm using windows 10 and have downloaded the V2.0 Alpha. 

I have set my configuration up as described in the read me. Using temp directory as my plotting SSD, final directory as one of my storage HDDs and adding a cache of 20G of RAM.

However when I try to start a plotting I get a fatal error, VirtualAlloc failed. Is this a bug or am I not understanding something about the config?

**Comment by harold-b** [comment, marked as answer]
`VirtualAlloc failed` means the OS was not able to give us the memory we asked it to give us to plot. At `64` buckets bladebit requires around 12G of RAM excluding any cache usage. So if you'd like to maximize cache usage you may try increasing the bucket count (the memory requirements get halved with each bucket count reduction) and lowering the amount assigned to cache.

**128** buckets are  not enabled in the alpha 2 build, [but you can download the latest development build which has it enabled](https://github.com/Chia-Network/bladebit/actions/runs/2966955989) (bottom of the page).  **128** buckets is a happy medium requiring around 6G of RAM, thus you can potentially assign more to cache without having to go down to 256 buckets.
